### PR TITLE
- adds .Net 6 sdk

### PR DESCRIPTION
**Pull request by baywet**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

When trying to install locally I'm getting a "invalid argument exception". Not sure where this is coming from. I'd appreciate if somebody could double check.

runing the installer with /install and /passive works

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34374)

### CODE REVIEW

File: manifests/m/Microsoft/dotnet/6.0.100/Microsoft.dotnet.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnet
+PackageVersion: 6.0.100
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnet/6.0.100/Microsoft.dotnet.installer.yaml:5]:
```suggestion
PackageVersion: 6.1.21.52711
```

File: manifests/m/Microsoft/dotnet/6.0.100/Microsoft.dotnet.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnet
+PackageVersion: 6.0.100
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnet/6.0.100/Microsoft.dotnet.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.1.21.52711
```

File: manifests/m/Microsoft/dotnet/6.0.100/Microsoft.dotnet.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnet
+PackageVersion: 6.0.100
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnet/6.0.100/Microsoft.dotnet.yaml:5]:
```suggestion
PackageVersion: 6.1.21.52711
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder from 6.0.100 to 6.1.21.52711 using Visual Studio Code by pressing the `.` (dot) on your keyboard on this PR.

**Comment by baywet** [comment]
Thanks for the suggestions. Yep I'm planning to do so but on my phone at the moment. Give me an hour.

**Comment by baywet** [comment]
done!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**